### DISCUSSION TITLE
`$schema` for suggesting auto-completions in settings.json

### DISCUSSION THREAD

**Discussion by nooreldeenmansour** [microsoft/AirSim#4183] [Ideas]
Hello,

Is there a JSON schema for the settings listed here https://microsoft.github.io/AirSim/settings/?, I'll gladly make a PR if there isn't a one.

Having a JSON schema would be great for the users of text-editors and IDEs that support auto-completions based on a schema, such as VSCode.

Regards,
Noor.

**Comment by jonyMarino** [comment]
@nooreldeensalah, AFAIK no one has made one. But, yes, it can be handy for users!
